src/test/java/edu/kis/powp/jobs2d/events/SelectClearMacroListener.java

```diff
@@ -0,0 +1,13 @@
+package edu.kis.powp.jobs2d.events;
+
+import edu.kis.powp.jobs2d.features.MacroFeature;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class SelectClearMacroListener implements ActionListener {
+    @Override
+    public void actionPerformed(ActionEvent actionEvent) {
+        MacroFeature.getMacroDriverDecorator().clearMemory();
```

Comment:
Have you checked if "ClearMacro" after "LoadMacro" affects the loaded macro? It shouldn't.